test/cri-containerd/stopcontainer_test.go

```diff
@@ -193,3 +193,48 @@ func Test_StopContainer_ReusePod_LCOW(t *testing.T) {
 	containerID = createContainer(t, client, ctx, request)
 	runContainerLifetime(t, client, ctx, containerID)
 }
+
+func Test_StopContainer_ZeroVPMEM_LCOW(t *testing.T) {
```

Comment:
Actually, why is this in `stopcontainer_test.go`?

Reply:
How about putting it in `container_test.go` and called something like `Test_RunContainer_ZeroVPMEM_LCOW`?

Reply:
Sounds good